lib/bitstyles_phoenix/bitstyles.ex

```diff
@@ -0,0 +1,64 @@
+defmodule BitstylesPhoenix.Bitstyles do
+  @moduledoc false
+
+  @default_version Application.compile_env!(:bitstyles_phoenix, :bitstyles_default_version)
+
+  def cdn_url do
+    case Application.get_env(:bitstyles_phoenix, :bitstyles_cdn) do
+      nil ->
+        nil
+
+      cdn ->
+        "#{cdn}@#{@default_version}"
+    end
+  end
+
+  @doc """
+  Returns the classnames for the configured version.
+  Input classnames are assumed to be from the #{@default_version} version of bitstyles.
+  """
+  def classname(name), do: classname(name, version())
+
+  def classname(class, version) when version > "2.0.0" do
```

Comment:
Technically this will break when bitcrowd reaches v10... But I wonder if we should actually indefinitely support all bitstyles versions

Reply:
Yes. and if that is the case I can still write a compare module for semvers 🗡️ 

I think i would occasionally drop versions as well. This is just meant for minor changes. Generally I want to build a version check on top of this that looks in the `node_modules` and warns when the wrong version is used.